guardian/scribe-plugin-noting
# Replace missing work, add es6, fix copied text bug.

This PR:

- Re-integrates work lost by https://github.com/guardian/scribe-plugin-noting/pull/49 (and my terrible merge).
- Adds es6 support.
- Removes Plumber from build.
- Adds a release script grabbed from scribe.
- Fixes https://github.com/guardian/scribe-plugin-noting/issues/53 which was re-introduced by `reset-note-barriers` refactoring.
- Adds `errorHandle` which adds debug information to thrown errors. 
- Adds `.jshintrc` to project

### QA
- Be sure to check that un-noting part of a note does not add any erroneous text:
![composer-unote-extra-text](https://cloud.githubusercontent.com/assets/1975139/5663111/6b28677a-973a-11e4-879d-892989e1a7b6.gif)
